### PR TITLE
dev, v2.1, v3.0: fix 3 broken links

### PR DESCRIPTION
**Pull request by CaitinChen**
### What is changed, added or deleted? 

Fix 3 broken links

### Which version does your change affect? 

dev, v2.1, v3.0

### CODE REVIEW

File: v3.0/releases/2.1ga.md [hunk 1]
```diff
@@ -253,7 +253,7 @@ On November 30, 2018, TiDB 2.1 GA is released. See the following updates in this
 
 + tikv-ctl
 
-    - [Add the `ldb` command to diagnose RocksDB related issues](https://github.com/tikv/tikv/blob/master/docs/tools/tikv-control.md#ldb-command)
+    - [Add the `ldb` command to diagnose RocksDB related issues](https://github.com/tikv/tikv/blob/master/docs/V2.1/tools/tikv-control.md#ldb-command)
```
Comment (lilin90) [inline v3.0/releases/2.1ga.md]:
```suggestion
    - [Add the `ldb` command to diagnose RocksDB related issues](https://github.com/tikv/tikv/blob/master/docs/reference/tools/tikv-control.md#ldb-command)
```

File: dev/releases/2.1ga.md [hunk 2]
```diff
@@ -252,7 +252,7 @@ On November 30, 2018, TiDB 2.1 GA is released. See the following updates in this
 
 + tikv-ctl
 
-    - [Add the `ldb` command to diagnose RocksDB related issues](https://github.com/tikv/tikv/blob/master/docs/reference/tools/tikv-control.md#ldb-command)
+    - [Add the `ldb` command to diagnose RocksDB related issues](https://github.com/tikv/tikv/blob/master/docs/V2.1/tools/tikv-control.md#ldb-command)
```
Comment (lilin90) [inline dev/releases/2.1ga.md]:
```suggestion
    - [Add the `ldb` command to diagnose RocksDB related issues](https://github.com/tikv/tikv/blob/master/docs/reference/tools/tikv-control.md#ldb-command)
```

**Comment by dcalvin** [comment]
I updated your edit directly with the link in tikv.org @lilin90

**Comment by lilin90** [comment]
@dcalvin Thanks! @CaitinChen Please address comments~

**Comment by CaitinChen** [comment]
@lilin90 PTAL again. Thanks.

**Comment by lilin90** [comment]
@CaitinChen One more comment to be addressed.

**Pull request merged**